lib/elixir/lib/kernel.ex

```diff
@@ -482,6 +482,26 @@ defmodule Kernel do
     :erlang.floor(number)
   end
 
+  @doc """
+  Returns its argument.
+
+  ## Examples
+
+      iex> identity(17)
+      17
+
+      iex> identity(:ok)
+      :ok
+
+      iex> 'abcdaabccc' |> Enum.sort |> Enum.chunk_by(&identity/1)
+      ['aaa', 'bb', 'cccc', 'd']
+
+  """
+  @spec identity(term) :: term
```

Comment:
```suggestion
  @spec identity(a) :: a when a: var
```

Reply:
I went ahead and applied the changes directly because they were quite straightforward :)